packages/ozone-typescript-client/src/itemClient/itemClientImpl.ts

```diff
@@ -77,6 +81,12 @@ export class ItemClientImpl<T extends Item> implements ItemClient<T> {
 		return this.client.call<SearchResults<FromOzone<T>>>(request)
 	}
 
+	graphQLSearch<TData = any, TVariables = OperationVariables>(query: TypedDocumentNode<TData, TVariables>, variables ?: TVariables): Promise<TData> {
+		return this.graphQLClient.query({
+			query: query, variables: variables
```

Comment:
Just so you know, this is equivalent to 
```suggestion
			query, variables
```